.github/contributing.md

```diff
@@ -75,6 +75,10 @@ test('?raw import', async () => {
 })
 ```
 
+## Debug Logging
+
+You can set the DEBUG environment variable to turn on debugging logs. E.g. `DEBUG="vite:resolve"`. To see all debug logs you can set `DEBUG="vite:*"`, but be warned that it will be quite noisy. You `grep` the code for `vite:` to see a list of available debug scopes.
```

Comment:
Could you rewrite the sentence:

> You `grep` the code for `vite:` to see a list of available debug scopes.

I (as a non native speaker) quite don't understand what you tried to tell with this. Or at least **how** to `grep`.
Maybe an example would help?
But I'm unsure and hope not you mean using e.g. `DEBUG="vite:*" yarn vite | grep -i "vite:"` or something like that. Cause that would be horrible and also don't show all possibilities. 😨

Reply:
done